src/Storefront/Test/Framework/Routing/StorefrontResponseTest.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Test\Framework\Routing;
+
+use PHPUnit\Framework\TestCase;
+use Shopware\Storefront\Framework\Routing\StorefrontResponse;
+
+/**
+ * @internal
+ */
```

Comment:
```suggestion
/**
 * @internal
 * @covers Shopware\Storefront\Framework\Routing\StorefrontResponse
 */
```

Please also add a covers annotation